NEWS.md

```diff
@@ -1,6 +1,7 @@
-
 ## 20.0.0 (in-progress)
 
+* Provides `declare-item-lost 0.3` (CIRC-1077)
```

Comment:
Done.

Reply:
I think I might have poorly explained what I meant. Rather than all of the changes making it into 20.1.0, I meant that the changes in this pull request would likely be in the next version and the others would be released prior to this change being merged.

```suggestion
## 20.1.0 (in-progress)

* Refund/cancel Aged to lost fees/fines when item Declared lost (CIRC-1077)
* Provides `declare-item-lost 0.3` (CIRC-1077)

## 20.1.0 (in-progress)

* No longer periodically executes age to lost background processes (CIRC-1084)
* Does not charge overdue fees when fee refund period has passed (CIRC-1000)
* Loan dates now truncate if there is a recall in any position in the item hold queue (CIRC-1018)
```